docs/ios/v2/features/screen-share.mdx

```diff
@@ -122,6 +124,15 @@ When a user leaves the meeting room from the main app, you will receive an error
         ...
     }
   ```
+  
+## Debugging issues in your screen-share implementation
+
+1. If you use cocoapods with M1 mac and compile for simulator then your might run into an error like this: "error framework not found Pods_hmsscreenshare"
+This is an issue if you use cocoapods for adding 100ms SDK and works fine if you are using swift package manager. As a workaround please use a snippet similar to at the end of this example podfile to exclude arm64 arch for simulator from your pod screen share target:
```

Comment:
Id simplify this a bit:
This is an issue if you use cocoapods for adding 100ms SDK and works fine if you are using swift package manager. => This issue is specific to cocoapods method of integrating the 100ms SDK.

Reply:
Rephrased but also left a mention of SPM just so people can switch of they want